### PR TITLE
Edited new Yomi section

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10653

### CODE REVIEW

File: modules/salt/pages/yomi.adoc [hunk 1]
```diff
@@ -4,24 +4,41 @@
 Yomi (yet one more installer) is an installer for {suse} and openSUSE operating systems.
 Yomi is designed as a SaltStack state, and can be used for installing {suse} operating systems on new systems.
 
-In the context of {productname} we can use Yomi as one step for the provisioning of a new system, and as an alternative to AutoYaST.
+In {productname}, Yomi can be used as part of provisioning new clients, or as an alternative to AutoYaST.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: I would remove the 'or', is an alternative to autoyast

File: modules/salt/pages/yomi.adoc [hunk 2]
```diff
@@ -4,24 +4,41 @@
...
 * Boot the new client using the PXE boot image for Yomi.
 
+
+
 == Install the ``yomi-formula``
 
-The first required component is the ``yomi-formula``, already available as a package in {productname}.
+Before you begin, you need to install the ``yomi-formula``, which is available as a package in {productname}.
+
+The ``yomi-formula`` package contains the Salt states and modules that describe the Yomi state, the {productname} formulas that will allow the creation of the pillar using the {productname} {webui}, the documentation about the different sections of the pillar, and some examples about how to parameterize installations based on openSUSE, MicroOS or SLE.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: > the {productname} formulas that will allow the creation of the pillar using the {productname} {webui}

I made a mistake: is not 'formulas that will', is "formula with forms that will"

File: modules/salt/pages/yomi.adoc [hunk 3]
```diff
@@ -4,24 +4,41 @@
...
+
+The formula package performs these actions:
+
+* Adds a new configuration file called ``yomi-formula.conf`` in the [path]``/etc/salt/master.d/`` directory.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: ... for finding the Python module and states required by Yomi.

I would comment the reason of this configuration change.

File: modules/salt/pages/yomi.adoc [hunk 4]
```diff
@@ -36,47 +53,63 @@ zypper in yomi-formula
 systemctl restart salt-master.service
 ----
 
-This package contains the Salt states and modules that describe the Yomi state, the {productname} formulas that will allow the creation of the pillar via the web interface, the documentation about the different sections of the pillar, and some examples about how to parameterize installations based on openSUSE, MicroOS or SLE.
 
-To find the new modules, this package is already adding a new configuration file in  [path]``/etc/salt/master.d``, named yomi-formula.conf.
+// For more information about the Yomi formula, see xref:salt:formula-yomi.adoc[] TBC --LKB 2020-03-03
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: Is this a doc that we need to provide too?

File: modules/salt/pages/yomi.adoc [hunk 5]
```diff
@@ -96,135 +129,150 @@ cobbler profile add \
...
-Set for User 1:
-* Username: root
-* Password Hash: $1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0
 
 
-== Monitoring the installation
+== Monitor the Installation
 
-Optionally we can monitor the installation (that will be commanded once we apply the highstate to the new system). For this the ``salt-api`` service is required.
+You can monitor the installation as it progresses, using the ``monitor`` tool from the ``salt-api`` service.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: 'monitor' is a tool from yomi, and for work requires the 'salt-api' service, from salt. I would phrase it like:

using the monitor too from Yomi. This tool requires the activation of the salt-api service. For more information about the ``salt-api`` service, and how to use the ``monitor`` tool that it provides, see https://github.com/openSUSE/yomi.

File: modules/salt/pages/yomi.adoc [hunk 6]
```diff
@@ -96,135 +129,150 @@ cobbler profile add \
...
+You can continue monitoring until the highstate is applied to the new client.
+You will need to have enabled ``Events`` in the Yomi formula.
 
-For instructions about how to install and configure this service, and how to use the provided ``monitor`` tool, please refer to the Yomi documentation. In order to instruct Yomi to send events every time that an installation step is produced, please, remember to select 'Events' in the first Yomi sub-form.
+For more information about the ``salt-api`` service, and how to use the ``monitor`` tool that it provides, see https://github.com/openSUSE/yomi.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: This could be moved up.

File: modules/salt/pages/yomi.adoc [hunk 7]
```diff
@@ -96,135 +129,150 @@ cobbler profile add \
...
-* Username: root
-* Password Hash: $1$wYJUgpM5$RXMMeASDc035eX.NbYWFl0
 
 
-== Monitoring the installation
+== Monitor the Installation
 
-Optionally we can monitor the installation (that will be commanded once we apply the highstate to the new system). For this the ``salt-api`` service is required.
+You can monitor the installation as it progresses, using the ``monitor`` tool from the ``salt-api`` service.
+You can continue monitoring until the highstate is applied to the new client.
```
Comment (aplanas) [inline modules/salt/pages/yomi.adoc]: The monitor is during the installation, so is during the application of the highstate

File: modules/salt/pages/yomi.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/salt/pages/yomi.adoc]: We will need to get the formula details into the Salt Guide with the other formulas. I'll create an issue so we don't forget about it.

Comment (Loquacity) [reply modules/salt/pages/yomi.adoc]: https://github.com/uyuni-project/uyuni-docs/pull/137#discussion_r391973906

File: modules/salt/pages/yomi.adoc (hunk 6 above)
Comment (Loquacity) [reply modules/salt/pages/yomi.adoc]: We like to keep the "for more ... see ..." content at the bottom of the section it applies to as much as possible.

File: modules/salt/pages/yomi.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/yomi.adoc]: I've split it up.

File: modules/salt/pages/yomi.adoc [hunk 8]
```diff
@@ -271,8 +273,9 @@ This behavior is set to be changed in future versions of Yomi.
 
 == Monitor the Installation
 
-You can monitor the installation as it progresses, using the ``monitor`` tool from the ``salt-api`` service.
-You can continue monitoring until the highstate is applied to the new client.
-You will need to have enabled ``Events`` in the Yomi formula.
+You can monitor the installation as it progresses, using the ``monitor`` tool from Yomi.
+You can continue monitoring as the highstate is applied to the new client.
```
Comment (keichwa) [inline modules/salt/pages/yomi.adoc]:
```suggestion
```
avoid single-sentence paras ;)

**Pull request merged**